pubsub/natspubsub/nats.go

```diff
@@ -23,6 +23,27 @@
 // for the scheme "nats".
 // The default URL opener will connect to a default server based on the
 // environment variable "NATS_SERVER_URL".
+//
+// For servers that support it (NATS Server 2.2.0 or later), messages could
```

Comment:
nit: could -> can

Reply:
Done.

Reply:
I don't see this change in the commit?

Reply:
Oh, I somehow didn't include it in the previous commit. Done.